vars/network_vars.yml

```diff
@@ -16,5 +17,24 @@ _wired_systemd_networkd_network:
         - Name: "{{ network_interface }}"
     - Network: "{{ _network_common + _vlan_conditionnal }}"
 
+_br0_systemd_networkd_netdev:
+  79-br0:
+    - NetDev:
+      - Name: "br0"
+      - Kind: "bridge"
+
+_br0_systemd_networkd_network:
+  79-br0:
+    - Match:
+      - Name: "br0"
+    - Network:
+      - Gateway: "{{ gateway_addr }}"
+      - Address: "{{ ip_addr }}/{{ subnet | default(24) }}"
+  79-wired:
+    - Match:
+      - Name: "{{ network_interface }}"
+    - Network: "{{ [ {'Bridge': 'br0'} ] | default([]) }}"
+
+systemd_networkd_network: "{{  _br0_systemd_networkd_network if create_br0_bridge is defined else _wired_systemd_networkd_network | combine(_systemd_networkd_network_custom) }}"
+systemd_networkd_netdev: "{{  _br0_systemd_networkd_netdev if create_br0_bridge is defined else [] | combine(_systemd_networkd_netdev_custom)}}"
```

Comment:
if create_br0_bridge is defined and create_br0_bridge

If you do not type "and create_br0_bridge", the bridge will be created even when create_br0_bridge is defined be set to false.